frankenstein/server.go

```diff
@@ -59,6 +59,11 @@ func AppenderHandler(appender storage.SampleAppender) http.Handler {
 					Value:     model.SampleValue(s.GetValue()),
 					Timestamp: model.Time(s.GetTimestampMs()),
 				})
+				if err != nil {
+					log.Errorf(err.Error())
+					http.Error(w, err.Error(), http.StatusBadRequest)
```

Comment:
Probably more like `http.StatusInternalServerError`, since at this point it's unlikely to be the client's fault?